packages/sdk-bridge/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ### Unreleased
 
+### 1.2.0-rc.0
+
+- (major) refactor: change message status to a tagged union instead of an enum,
```

Comment:
have we released any `1.x.x` version?

Reply:
if not, no need to bump to `1.2.0`. we can still put major changes in label changes

Reply:
Latest release is `1.1.0-rc.0`

Reply:
according to npm, there has been no `1.x.x` version without a label. so we're still pre-1.0.0, technically speaking

Reply:
What are you looking at? It shows it's been published...
<img width="420" alt="Screen Shot 2022-07-26 at 12 29 55 PM" src="https://user-images.githubusercontent.com/36751902/181084281-beb7f260-8b17-4ed9-99b7-3197bff92dc5.png">

Reply:
Oh, "without a label". So, what are you suggesting?

Reply:
`major.minor.patch-label`

A version with a label (anything after a hyphen) is NOT a version for the purposes of semver, and there's no requirement to respect semver 

We need to stop publishing labelers versions, but since we have (so far) ONLY done labeled versions, anything goes

Reply:
ok... yeah, they were planning on doing away with labels soon, but it hasn't happened yet and I just need these fixes out. Reference [this ticket](https://app.clickup.com/t/14205595/ENG-1336) and then just tell me what major versions you want and I'll just get it done

Reply:
Updated to `2.0.0` for sdk and `1.0.0` for multi-provider, sdk-bridge and sdk-govern